swagger/v3/api_spec.json

```diff
@@ -4687,7 +4687,6 @@
               "moved-school",
               "mentor-no-longer-being-mentor",
               "school-left-fip",
-              "started-in-error",
```

Comment:
could you remove this value from all API versions please?

Reply:
Hi @cwrw tks for your review.. I can't find any more occurrences of `started-in-error` for ECF related stuff.